frontend/pages/admin/UserManagementPage/components/UserForm/UserForm.tsx

```diff
@@ -524,7 +526,9 @@ const UserForm = ({
         className={`${baseClass}__radio-input`}
         label="Password"
         id="password-authentication"
-        disabled={!(smtpConfigured || sesConfigured)}
+        // allow the user to change auth back to password if they only changed the form to SSO in
+        // the current session, that is, in the db, the user is still password authenticated
+        disabled={!(smtpConfigured || sesConfigured) && !initiallyPasswordAuth}
```

Comment:
This means once a user is set to use SSO, they can never switch to password?  Is that what we settled on?  I thought we were debating how to avoid saving a user with no password, which would require more backend work.  But they could also just use "reset password" in that case.

Reply:
No, it means once a user is set to use SSO, they can (from the UI) only be switched to password if SES or SMTP is configured for the org. This is how the form behaved previously.

This change removes disabling the option if the user is just poking different options in the form and hasn't saved anything yet.

I asked @rachaelshaw about this stuff in [this slack thread](https://fleetdm.slack.com/archives/C084F4MKYSJ/p1736537341715539), Rachael can you please respond there, thanks

Reply:
Actually, it's probably best to have the conversation here – will paste in comment thread

Reply:
I think I just misread the boolean, apologies.  Looks good.